### DISCUSSION TITLE
Web3auth User Flow Screens

### DISCUSSION THREAD

**Discussion by bt-latricia** [Web3Auth/Web3Auth#965] [Help]
I need help changing the user flow screen. I would like to change the logo and the name. I need help making those changes within an app. The trouble that I am having is that I can't identify where the current default setting is located.

**Comment by Saviour1001** [comment]
Depending upon the SDK you are using in your App, you can add Whitelabel settings in it which allows you to change things like `appLogo`, `theme` etc. 
If you are using `@web3auth/modal` here is the [Whitelabel section](https://web3auth.io/docs/sdk/web/modal/whitelabel), similarly we have one in other SDKs as well. The docs contain a detailed description of the parameters one can pass in.
